README.rst

```diff
@@ -980,7 +985,6 @@ may want to familiarize with:
   the software that powers PyPI_ itself. It is not generally intended to
   be run by end-users.
 
-
```

Comment:
I am not a big expert on RST syntax and also prefer markdown myself, but, for consistency, other headings with `===` level also have two blank lines before them, maybe it's better to keep this one

Reply:
Sure.

Reply:
FWIW pypi does support markdown now, provided you specify a config value in the setup.py, so it might be worth switching this to markdown at some point. I think very few people ever have preferred RST :)